packages/vuetify/src/presets/default/index.ts

```diff
@@ -13,6 +13,8 @@ import { VuetifyPreset } from 'vuetify/types/services/presets'
 
 export const preset: VuetifyPreset = {
   breakpoint: {
+    // TODO: update to MD2 spec in v3 - 1280
+    mobileBreakPoint: 1264,
```

Comment:
Should the tests also be testing 1264 as opposed to 1280 if this is the default until v3?